### DISCUSSION TITLE
iOS Reconsidering Projects

### DISCUSSION THREAD

**Discussion by Woshimbo** [community/community#49904] [Mobile]
So after checking out Projects on the web browser (Can not add anything to a board, such as a to do list.) and on the app. (Which doesn't allow actual modification or adding to the projects.) 

It seems like this isn't too valuable for people trying to add and do typing from their phone sadly. Hopefully it will all start working soon.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.103.0 (93212509)
Device: iPhone 12 (iOS 16.3.1)
</details>

**Comment by stevepopovich** [comment]
Hi @RollingBytes! Yes, we agree it would be valuable to be able to add new items. We are planning to add this in future. The exact timeline is TBD, but keep an eye out for future updates for this!
